charts/testkube-cloud-api/templates/deployment.yaml

```diff
@@ -302,6 +307,10 @@ spec:
               value: "{{ $minioSkipVerify }}"
             - name: MINIO_EXPIRATION
               value: "{{ .Values.api.minio.expirationPeriod }}"
+            {{- if .Values.api.minio.credsFilePath }}
+            - name: MINIO_SHARED_CREDENTIALS_FILE
```

Comment:
was this env var added to the API code? also how to define the creds in this file?